src/coreComponents/mesh/docs/Mesh.rst

```diff
@@ -184,58 +184,97 @@ The mesh block has the following syntax:
   <Mesh>
     <VTKMesh
       name="MyMeshName"
+      logLevel="1"
       file="/path/to/the/mesh/file.vtk"/>
   </Mesh>
 
-We advise users to use absolute path to the mesh file.
+We advise users to use absolute path to the mesh file, and strongly recommend the use of a logLevel
+of 1 or more to obtain some information about the mesh import. This information contains for 
+example the list of regions that are imported with their names, which is particularly useful to 
+fill the ``cellBlocks`` field of the ``ElementRegions`` block (see below). Some information about the 
+imported surfaces is also provided.
 
 GEOSX uses ``ElementRegions`` to support different physics
 or to define different constitutive properties.
-An ``ElementRegion`` is defined as a set of ``CellBlocks``.
+The ``ElementRegions`` block can contain several ``CellElementRegion`` blocks. A ``CellElementRegion`` is defined as a set of ``CellBlocks``.
 A ``CellBlock`` is an ensemble of elements with the same element geometry.
 
+The naming of the imported ``cellBlocks`` depends on whether the data array called ``regionAttribute`` is
+present in the vtu file or not. This data array is used to define regions in the vtu file and
+assign the cells to a given region. The ``regionAttribute`` is a integer and not a string
+(unfortunately).
+
 .. figure:: mesh_multi.png
    :align: center
    :width: 500
 
 In the example presented above, the mesh is is composed of two regions (*Top* and *Bot*).
-Each region contains 3 ``CellBlocks``.
+Each region contains 4 ``cellBlocks``.
 
-The ``ElementRegions`` are defined as below :
+- If the vtu file does not contain ``regionAttribute``, then all the cells are grouped in a single
+region, and the cell block names are just constructed from the cell types (hexahedra, wedges,
+tetrahedra, etc). Then in the exemple above, the ``ElementRegions`` can be defined as bellow:
```

Comment:
You can visualize the documentation generated by ReadTheDocs here: https://geosx-geosx--2316.com.readthedocs.build/en/2316/coreComponents/mesh/docs/Mesh.html

To find this link, you just have to click on the "green check" on your PR (the thing that becomes a "red cross" if the test don't pass) and click on "details" and then "View docs" on the ReadTheDocs website.

You will see that the indentation is slightly messed up in a few places, if you don't mind fixing it. Let me know if you don't find the link